presentation/src/main/java/com/mafraq/presentation/features/home/components/ItemResultSearch.kt

```diff
@@ -0,0 +1,58 @@
+package com.mafraq.presentation.features.home.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Card
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mafraq.presentation.R
+import com.mafraq.presentation.design.components.Spacer
+import com.mafraq.presentation.design.theme.MafraqTheme
+import com.mafraq.presentation.utils.extensions.painter
+
+@Composable
+fun InfoCard(
+    headerText: String,
+    bodyText: String,
+) {
+    Box(Modifier.fillMaxWidth()) {
+
+        Card {
+            Row(verticalAlignment = Alignment.CenterVertically) {
+                Icon(
+                    painter = R.drawable.ic_location_background.painter,
```

Comment:
- Use the Icon only without the the white background circle

- The name should `be ic_location_filled`